.dockerignore

```diff
@@ -1,3 +1,4 @@
 tmp
 log
-.git
\ No newline at end of file
+.git
+node_modules
```

Comment:
was this slowing us down before?

Reply:
not slowing us down, but node_sass built locally in darwin was overwriting the one built in the docker image